packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -1077,6 +1081,28 @@ export function errorHydratingContainer(parentContainer: Container): void {
   }
 }
 
+export function warnOnHydrationMismatch(
+  fiber: Fiber,
```

Comment:
Fibers should never leak into the DOM host config. It currently leaks in a few places in the event system but that's isolated to that and the plan is to get rid of those synthetic parts eventually anyway.

Think of it as I should be able to write a completely different implementation of Fiber that doesn't even use Fibers and still reuse the host config. Which we likely will.

Reply:
I noticed that you didn't actually add the first Fiber. The test selectors seems to have done it too. We'll have to go back to reaffirm the abstraction boundary with those too.